v63003/outref/gtm8844.txt

```diff
@@ -0,0 +1,85 @@
+# Running functions that violate our restrictions
+# -------------------------------------------------------------------------------------------
+# Two consecutive halts
+# Halt 1, Expect a RESTRICTEDOP error
+ZSTATUS=haltfn+3^gtm8844,%YDB-E-RESTRICTEDOP, Attempt to perform a restricted operation: HALT
+# Halt 2
+%YDB-F-RESTRICTEDOP, Attempt to perform a restricted operation: HALT
+##TEST_AWKYDB_FATAL_ERROR.ZSHOW_DMP_[0-9]*_1.txt
+# -------------------------------------------------------------------------------------------
+# Two consecutive zhalts
+# Zhalt 1, Expect a RESTRICTEDOP error
+ZSTATUS=zhaltfn+3^gtm8844,%YDB-E-RESTRICTEDOP, Attempt to perform a restricted operation: ZHALT
+# Zhalt 2
+%YDB-F-RESTRICTEDOP, Attempt to perform a restricted operation: ZHALT
+##TEST_AWKYDB_FATAL_ERROR.ZSHOW_DMP_[0-9]*_1.txt
+# -------------------------------------------------------------------------------------------
+# Two consecutive halts with a pause
+# Halt 1, Expect a RESTRICTEDOP error
+ZSTATUS=haltfn+3^gtm8844,%YDB-E-RESTRICTEDOP, Attempt to perform a restricted operation: HALT
+# Halt 2
```

Comment:
I was hoping to see a RESTRICTEDOP error even for this but no process termination so we should proceed and print the "Haltfn failed" line. But I don't see that in the output here. Do you have an explanation for this?